### PR TITLE
chore: className for compass plugin

### PR DESCRIPTION
**Pull request by venmax**
**Merge request checklist**

-   [x] All lints and tests pass. If needed, new unit tests were added.
-   [ ] If needed, the [documentation](https://github.com/mistic100/Photo-Sphere-Viewer/tree/main/docs) has been updated.

ClassName for compass. for example `position: [string, string]` may not ehough for positioning

### CODE REVIEW

**Comment by mistic100** [comment]
Please add a comment on the property and update the documentation

File: packages/compass-plugin/src/CompassPlugin.ts [hunk 1]
```diff
@@ -15,6 +15,7 @@ const getConfig = utils.getConfigParser<CompassPluginConfig, ParsedCompassPlugin
         navigationColor: 'rgba(255, 0, 0, 0.2)',
         hotspots: [],
         hotspotColor: 'rgba(0, 0, 0, 0.5)',
+        className: ''
```
Comment (mistic100) [inline packages/compass-plugin/src/CompassPlugin.ts:18]:
```suggestion
        className: null,
```

**Review by mistic100 — CHANGES_REQUESTED** [review]

**Comment by venmax** [comment]
> Please add a comment on the property and update the documentation

done

File: docs/plugins/compass.md [hunk 2]
```diff
@@ -201,6 +201,12 @@ Color of the navigation cone.
 
 Default color of hotspots.
 
+#### `className`
+
+-   type: `string`
```
Comment (mistic100) [inline docs/plugins/compass.md:206]:
```suggestion
-   type: `string`
-   updatable: yes
```

**Review by mistic100 — CHANGES_REQUESTED** [review]

**Pull request merged**